e2e/tests/v2/test_volume.robot

```diff
@@ -0,0 +1,53 @@
+*** Settings ***
+Documentation    v2 Data Engine Test Cases
+
+Test Tags    regression    v2
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/storageclass.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/setting.resource
+Resource    ../keywords/node.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Test V2 Volume Basic
+    [Tags]  coretest
+    [Documentation]    Test basic v2 volume operations
+    When Create volume 0 with    dataEngine=v2
+    And Attach volume 0
+    And Wait for volume 0 healthy
+    And Write data to volume 0
+    Then Check volume 0 data is intact
+    And Detach volume 0
+    And Delete volume 0
+
+Degraded Volume Replica Rebuilding
+    [Tags]    coretest
+    Given Disable node 2 scheduling
+    And Create storageclass longhorn-test with    dataEngine=v2
```

Comment:
nit: Storage class name could be prefix by `sc` like `sc-longhorn-default`

Reply:
I'm not sure about this. The build-in storage class is `longhorn (default)` instead of the one we creates:

```
$ kubectl get sc
NAME                   PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
longhorn (default)     driver.longhorn.io      Delete          Immediate              true                   11m
```

And if we want to use this `longhorn (default)` storage class, we simply write:

```
And Create persistentvolumeclaim 0 using RWO volume with longhorn storageclass
``` 

This does't need the prefix `sc` either.